rules/Arguments/ArgumentDefaultValueReplacer.php

```diff
@@ -31,13 +35,53 @@ public function processReplaces(
             if (! isset($node->params[$replaceArgumentDefaultValue->getPosition()])) {
                 return null;
             }
+
+            $this->processParams($node, $replaceArgumentDefaultValue);
```

Comment:
I think it should be `return $this->processParams($node, $replaceArgumentDefaultValue)` instead, same with `return $this->processArg($node, $replaceArgumentDefaultValue)` in the elseif to avoid issue repetitive code that solved by return in the method call when possible, see https://github.com/rectorphp/rector-src/pull/867/files#diff-4d0c9d1c076403126c86dce65d87ddfb14201065c02452a53e8ada699ddc3abe

so, in the `processParams()` and `processArgs()`, should be return null or return $classMethod/$expr.

Reply:
Fixed.